### PR TITLE
Update about

### PR DESCRIPTION
**Pull request by bjornorri**
Fix embarrassing outdated info.

### CODE REVIEW

**Comment by kthelgason** [comment]
Thanks for the PR. I'd like to work with you to land this! 

Unfortunately this PR does not go far enough in addressing out-of-date information, as an example, I no longer live in Reykjavik. Additionally, while I appreciate the sentiment, I don't think Google needs an explanatory link.

File: about.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ I'm Kári, a software developer in Reykjavik, Iceland. I enjoy
 * Crossfit
 * Running
 * Biking
-* Cooking
+* Cocking
```
Comment (kthelgason) [inline about.md:13]: Please fix this spelling mistake

File: about.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ I'm Kári, a software developer in Reykjavik, Iceland. I enjoy
...
 * Reading
 
-I currently work at [OZ](www.oz.com) as a software engineer.
+I currently work at [Google](https://google.com) as a software engineer.
 I can be found on Twitter as [@karitristan](https://twitter.com/karitristan) and as
 kthelgason *at* google mail *dot* com.
```
Comment (kthelgason) [inline about.md:18]: While not specifically related to this PR, could you please remove the email, as I no longer want it visible.

**Review by kthelgason — COMMENTED** [review]
Thanks for your contribution 😄

File: about.md (hunk 1 above)
Comment (bjornorri) [reply about.md:13]: It's not.

**Comment by kthelgason** [comment]
Closing due to inactivity.

**Pull request closed without merging**